### DISCUSSION TITLE
Projected job time VS preview estimated time

### DISCUSSION THREAD

**Discussion by ronisegoly** [arkypita/LaserGRBL#1223] [Q&A] [answered]
Hi
I just replaced my device so it's brand new and I did not change anything in setting or board configuration
When I engrave I need to set the speed and I set it to 2000 mm/min
But when machine start moving the projected time is changed considerably ( like from 1 hour to 6)
Any idea?

**Comment by arkypita** [comment, marked as answer]
Before start the time is estimated by set speed and total job lenght.
When running the estimated time is projected by the actual, real, speed of the engraver.

So, maybe your engraver is not moving at the speed you set in job.
This could be because you have a acceleration or max speed limit in your engraver configuration, so the engraver cannot reach the settled speed.

Please read: http://lasergrbl.com/wp-content/uploads/2020/05/Grbl-Configuration-ENG.pdf

**Comment by arkypita** [comment]
However this is not a LaserGRBL issue, so it should be posted under "[discussions](https://github.com/arkypita/LaserGRBL/discussions)" (moved)

**Comment by ronisegoly** [comment]
thanks
can you explain why engraving image A takes less time than image B? Same parameters and size more or less
[A](https://www.dropbox.com/s/jg51t0f581g2v62/Depositphotos_53189361_xl-2015.jpg?dl=0)
[B](https://www.dropbox.com/s/gc4wwlpd8xw74pa/path21.png?dl=0)

**Comment by arkypita** [comment]
Because the total path to follow is shorter
